classes/item.rb

```diff
@@ -0,0 +1,41 @@
+class Item
+  attr_accessor :author, :source
```

Comment:
- [ ] I think `attr_accessor` not only has two (:authors, :source) items. There are two items missing (:genre, and :labels). Kindly add them as well. 

> HINT:
> You also make them only:
> attr_reader :author
> attr_reader :source
> ...
>